### PR TITLE
Upgrade to Play 2.7.0-RC9

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by ignasi35** [comment]
related to #111

File: conf/application.conf [hunk 1]
```diff
@@ -11,7 +11,7 @@ play {
 
   http {
     requestHandler = play.api.http.DefaultHttpRequestHandler
-    secret.key = changeme
+    secret.key = a-long-secret-to-calm-the-rage-of-the-entropy-gods
```
Comment (ignasi35) [inline conf/application.conf]: 🙇🏼‍♂️

File: app/actors/DocumentationPollingActor.scala [hunk 2]
```diff
@@ -33,7 +32,9 @@ object DocumentationPollingActor {
  * documentation.
  */
 class DocumentationPollingActor @Inject() (messages: MessagesApi, @Assisted repos: DocumentationGitRepos,
-                                           @Assisted documentationActor: ActorRef) extends Actor {
+                                           @Assisted documentationActor: ActorRef) extends Actor
+    with ActorLogging
+{
```
Comment (ignasi35) [inline app/actors/DocumentationPollingActor.scala]: Formatting is a bit off. Not a blocker, though.

File: app/actors/DocumentationPollingActor.scala [hunk 3]
```diff
@@ -108,7 +109,7 @@ class DocumentationPollingActor @Inject() (messages: MessagesApi, @Assisted repo
             implicit val lang = repos.default.config.lang
 
             if (old.isDefined) {
-              Logger.info(s"Updating default documentation for $version: $cacheId")
+              log.info(s"Updating default documentation for {}: {}", version, cacheId)
```
Comment (ignasi35) [inline app/actors/DocumentationPollingActor.scala]:
```suggestion
              log.info("Updating default documentation for {}: {}", version, cacheId)
```

File: app/services/github/ContributorsSummariser.scala [hunk 4]
```diff
@@ -86,15 +86,17 @@ class DefaultContributorsSummariser @Inject() (gitHub: GitHub, config: GitHubCon
...
 
   actorSystem.scheduler.schedule(0.seconds, 24.hours) {
     delegate.fetchContributors.onComplete {
-      case Failure(t) => Logger.error("Unable to load contributors from GitHub", t)
+      case Failure(t) => log.error("Unable to load contributors from GitHub", t)
       case Success(cs) =>
         if (contributors != cs) {
           val count = contributors.committers.size + contributors.playOrganisation.size + contributors.contributors.size
-          Logger.info("Loaded " + count + " contributors for GitHub")
+          log.info(s"Loaded $count contributors for GitHub")
```
Comment (ignasi35) [inline app/services/github/ContributorsSummariser.scala]:
```suggestion
          log.info("Loaded {} contributors for GitHub", count)
```

File: app/actors/DocumentationPollingActor.scala (hunk 2 above)
Comment (dwijnand) [reply app/actors/DocumentationPollingActor.scala]: I dislike the "align at a random column" formatting but I also avoid pushing my own opinions on formatting in unrelated code.  So this is my attempt to mixing in a trait that would be too long for the same line.

File: app/services/github/ContributorsSummariser.scala (hunk 4 above)
Comment (dwijnand) [reply app/services/github/ContributorsSummariser.scala]: Thanks!  I forgot SLF4J supported this.

**Pull request merged**